design/FY2022/NFP-Latent_Sizing.md

```diff
@@ -0,0 +1,592 @@
+
+    Temp = CalcZoneSizing(CurOverallSimDay, ControlledZoneNum).CoolDesTemp;
+    HumRat = CalcZoneSizing(CurOverallSimDay, ControlledZoneNum).CoolDesHumRat;
+    DeltaTemp = Temp - Node(ZoneNode).Temp;
+    Enthalpy = PsyHFnTdbW(Temp, HumRat);
+    SysOutputProvided = ZoneSysEnergyDemand(ActualZoneNum).RemainingOutputRequired;
+    CpAir = PsyCpAirFnW(HumRat);
+    MassFlowRate = max(SysOutputProvided / (CpAir * DeltaTemp), 0.0);
+
+    UpdateSystemOutputRequired(state, ActualZoneNum, SysOutputProvided, LatOutputProvided);
+
+These data are placed on the zone supply node for use in the correct step of `ZoneTempPredictorCorrector` to calulate the resulting zone temperature and humidity ratio. In current EnergyPlus calculations, the zone temperature is maintained at the thermostat set point temperature while the zone humidity ratio floats according to the humidity ratio entering the zone at the zone supply node **sensible** mass flow rate. It is not important at this time to actually provide the zone the correct mass flow rate, what is important here is to meet the zone sensible load exactly such that the zone temperature remains at the thermostat set point.
+
+    Node(SupplyAirNode).Temp = Temp;
+    Node(SupplyAirNode).HumRat = HumRat;
+    Node(SupplyAirNode).Enthalpy = Enthalpy;
+    Node(SupplyAirNode).MassFlowRate = MassFlowRate;
+
+Finally, save the time series data. The example shown here is when there is a cooling load. This is actually incorrect since the air loop may not peak at the same time as the zone and the documentation for zone conditions for both heating and cooling need to be known for each time step (e.g., should not set `HeatZoneTemp = 0`). This data saving methodoloty will be revised during this effort (i.e., HeatZoneTemp and CoolZoneTemp will always be saved regardless of load).
+
+    CalcZoneSizing(CurOverallSimDay, ControlledZoneNum).HeatLoad = SysOutputProvided;
+    CalcZoneSizing(CurOverallSimDay, ControlledZoneNum).HeatMassFlow = MassFlowRate;
+    CalcZoneSizing(CurOverallSimDay, ControlledZoneNum).HeatZoneTemp = Node(ZoneNode).Temp;
+    CalcZoneSizing(CurOverallSimDay, ControlledZoneNum).HeatZoneHumRat = Node(ZoneNode).HumRat;
+    // why do we set these to 0 in E+, doesn't the allocation initialization do that?
+    CalcZoneSizing(CurOverallSimDay, ControlledZoneNum).CoolLoad = 0.0;
+    CalcZoneSizing(CurOverallSimDay, ControlledZoneNum).CoolMassFlow = 0.0;
+    CalcZoneSizing(CurOverallSimDay, ControlledZoneNum).CoolZoneTemp = 0.0;
+    CalcZoneSizing(CurOverallSimDay, ControlledZoneNum).CoolZoneHumRat = 0.0;
```

Comment:
These equations are misleading. Should have shown cooling equations (e.g., CoolLoad = -SysOutputProvided and CoolMassFlow = MassFlowRate). Also disregard comment about equations being incorrect, that was a mistake during formulation of these steps. Nothing needs to change in this area of code.